docs/conf.py

```diff
@@ -128,7 +147,7 @@
 # [howto, manual, or own class]).
 latex_documents = [
     (master_doc, 'minos.tex',
-     'Minos Microservice SAGA Documentation',
+     'Minos Microservice Common Documentation',
```

Comment:
Shouldn't this be `Saga`?
```suggestion
     'Minos Microservice Saga Documentation',
```